trade-finance-manager-ui/server/controllers/utilisation-reports/add-to-an-existing-payment/index.test.ts

```diff
@@ -212,11 +214,71 @@ describe('controllers/utilisation-reports/add-to-an-existing-payment', () => {
       expect((res._getRenderData() as AddToAnExistingPaymentViewModel).reportId).toEqual('123');
     });
 
+    it('should set the back link href', async () => {
+      // Arrange
+      const { req, res } = httpMocks.createMocks({
+        session: requestSession,
+        params: { reportId },
+        body: {
+          'feeRecordIds-1,22-reportedPaymentsCurrency-USD-status-TO_DO': 'on',
+          'feeRecordIds-333-reportedPaymentsCurrency-USD-status-TO_DO': 'on',
```

Comment:
Would be nice to use a constant for the `TO_DO` status, I think it was mentioned this is a bigger/separate piece though

Reply:
Yeah I think we should defer this to another ticket as it would require a lot of replacements